presale/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    'name': "Presale Orders",
+    'version': '1.0.0',
+    'depends': ['sale_management'],
+    'data': [
+        'security/ir.model.access.csv',
+        'security/presale_security.xml',
+        'views/presale_order_views.xml',
+        'views/sale_order_views.xml',
+        'views/presale_menus.xml',
+        'data/presale_order_sequence.xml',
+        'data/email_templete.xml',
+        'data/archive_presale_orders_cron.xml'
+    ],
+    'author': "Moaz Elmaatawy (elmo)",
```

Comment:
We'll usually keep `Odoo PS` as the author as multiple members of the department might be working on the same modules